lib/qa/authorities/assign_fast/generic_authority.rb

```diff
@@ -53,7 +53,7 @@ def build_query_url(q)
       #   See oclc sample code at
       #   http://experimental.worldcat.org/fast/assignfast/js/assignFASTComplete.js
       def clean_query_string(q)
-        URI.escape(q.gsub(/-|\(|\)|:/, ""))
+        ERB::Util.url_encode(q.gsub(/-|\(|\)|:/, ""))
```

Comment:
I think `ERB::Util.url_encode` will properly encode all of those except for the dash.  I tested it in IRB really quick
```
2.3.6 :003 > ERB::Util.url_encode("-|\(|\)|:")
 => "-%7C%28%7C%29%7C%3A"
```
I am not really sure  how this is being used elsewhere, so I am not sure what removing the gsub would potentially break.  Allowing the special characters may improve the autosuggest functionality slightly by allowing users to search for terms containing special characters